### PR TITLE
Campaign finder  refactoring

### LINKED ISSUE #2817: [Campaign Finder] Refactor campaign tiles to use the new establish Neue patterns.
The gallery and campaign tiles on the home page are using old code. Need to update them to use the new gallery pattern and the campaign tile pattern.

### PR DESCRIPTION
**Pull request by weerd**
Resolves #2817  

@DFurnes 
Further cleanup on old finder styles will happen post PR merge ;)

### CODE REVIEW

File: lib/themes/dosomething/paraneue_dosomething/js/finder/ResultsView.js [hunk 1]
```diff
@@ -28,13 +31,14 @@ define(function(require) {
 
     /**
      * Construct the cloneables and set the parent div where the results go
-     * @param  {jQuery} $div The div where the results will live
-     * @param  {jQuery} $blankSlateDiv The div containing initial pre-filtered state
-     * @param  {function} paginateAction Function to query next page of results.
+     * @param  {jQuery} $container        The container element where the results will live
+     * @param  {jQuery} $blankSlateDiv    The div containing initial pre-filtered state
```
Comment (DFurnes) [inline lib/themes/dosomething/paraneue_dosomething/js/finder/ResultsView.js:35]: Might make sense to rename this `$blankSlate` as well.

Comment (weerd) [inline lib/themes/dosomething/paraneue_dosomething/js/finder/ResultsView.js:35]: or `$blankSlateContainer` ?

Comment (DFurnes) [inline lib/themes/dosomething/paraneue_dosomething/js/finder/ResultsView.js:35]: Yeah, that makes sense.

**Comment by DFurnes** [comment]
Looks good! :sun_with_face:

**Pull request merged**